components/README.md

```diff
@@ -0,0 +1,141 @@
+[TOC]
+
+
+
+# Dialogs
+
+Dialogs are a standard way to inform the user about some result, giving the user option to react to it.
+
+<details>
+    <summary>Code snippet</summary>
```

Comment:
suggestion: I think it would be more beneficial for someone who read this documentation to read usage, then implementation of a widget. So you can link @Preview, rather then implementation.

Reply:
good idea 🆗